config/dataplane/dataplane.yaml

```diff
@@ -4,22 +4,27 @@ metadata:
   name: dataplane
   namespace: system
   labels:
-    app: dataplane
+    app: blixt
+    component: dataplane
 spec:
   selector:
     matchLabels:
-      app: dataplane
+      app: blixt
+      component: dataplane
   template:
     metadata:
       labels:
-        app: dataplane
+        app: blixt
+        component: dataplane
     spec:
-      serviceAccountName: udproute-manager
       hostNetwork: true
       containers:
       - name: dataplane
         image: ghcr.io/kong/blixt-dataplane:latest
         securityContext:
           privileged: true
-        args: ["eth0"]
+        args: ["-i", "eth0"]
```

Comment:
Should this be configurable?

Reply:
Eventually I expect we may remove this argument entirely and allow the loader to determine the interfaces it needs to deploy ebpf programs based on some dynamic criteria. For the moment, this just fixes something that was already broken, I think once we get to #45 we'll start having a better idea about how this is gonna look going forward.